### ISSUE TITLE
No sounds dumped from Trap Adventure 2 (iOS)

### ISSUE THREAD

**Issue by EmK530** [LAK132/SourceExplorer#149]
### File tested (Application.cci)
https://drive.google.com/file/d/1oiH3v4B62LGSf-V8JDZLsGbWnAGKCtUi/view

Dropping the file into Source Explorer does properly dump all images, but no sounds.

On line 90052 in the crash log, notice the ":TODO: HACK FIX FOR CCN SOUND BANK"
I would assume something isn't finished here?

### Crash log

[ATTACH-TO-ISSUE-ON-SOURCE-EXPLORER-GITHUB-REPO.txt](https://github.com/user-attachments/files/16454399/ATTACH-TO-ISSUE-ON-SOURCE-EXPLORER-GITHUB-REPO.txt)

**Comment by KuroFox02** [comment]
Yeah thats right i tryied to rip the sounds me too. and don't works for the sounds.

**Comment by KuroFox02** [comment]
i found an other program for ripping the sound effects.
(Game Extractor)

There are the TrapAdventure 1 & 2 SFX Pack
https://drive.google.com/file/d/1Jkh6UIV8as6c34ssUszCz4xThaLNTE82/view?usp=drive_link

**Comment by EmK530** [comment]
Hi I appreciate the help but I already managed to get the sound effects & music long ago using NebulaFD.
This is just an issue I was encouraged to post on here by LAK132, who then just hasn't responded to this.
